### ISSUE TITLE
Mehrzweck-Wege

### ISSUE THREAD

**Issue by hungerburg** [openstreetmap/iD#7592]
I know, this name is in the OSM wiki, but "Mehrzweck-Weg" is really a bad translation of "path" - The consequence being, that anybody native German will never use it for anything but a "track", and use "footway" instead for hiking paths &c. Can that be changed in ID? I suggest "Pfad" instead ("Weg" is already too broad, as it includes even roads).

**Comment by manfredbrandl** [comment]
I did change the Wiki because you are absolutely right that "Mehrweck-Weg" is a bad translation for "path", "Pfad" is much better. Please reviewe the Wiki if i have done it right. If the Change in the Wiki persists we can change the translation in iD via transifex.

**Comment by hungerburg** [comment]
Hallo Manfred, in the context there this now looks over the top. Maybe "Weg" is better, at least without the stupid "Mehrzweck" (multipurpose) addon. For reference, the dictionary translation of path is indeed way https://dict.leo.org/englisch-deutsch/path - while the reverse https://dict.leo.org/englisch-deutsch/pfad is much more narrow. I try my hands on the wiki, please check quality

Perhaps "Fußwege" to "Fußgängerwege" (how to gender?) so this gets clear?

**Comment by hungerburg** [comment]
Hmm, Fußwege is not in the Wiki…

**Comment by manfredbrandl** [comment]
I did change the Wiki again, please check quality. I also updated the translation for iD in Transifex.

**Comment by hungerburg** [comment]
Ja, so ist es viel besser.

How about "Gehweg" for "Fußweg" in iD, this is much more concise. Then I think this will be much harder to be gotten wrong by newbies and this issue ripe for closing!

**Comment by manfredbrandl** [comment]
I checked how many results Google has for Fußweg (12,6 Mio) and for Gehweg (4,6 Mio). Also Gehweg is Not known in Switzerland, therefore I stick to Fußweg.

**Comment by hungerburg** [comment]
Wikipedia is clear on the official usage - https://de.wikipedia.org/wiki/Gehweg is the word for what is meant with footway on OSM in Germany and Austria. Regarding the numbers: Gehweg is mostly used that way, unlike Fußweg, which can appear in other contexts too, eg. "Es war ein langer Fußweg von A nach B". "Fussweg", which is the official word for "Gehweg" in Switzerland only has 733000 hits on google, including all the other meanings, eg. "Einen Fussweg entfernt von …"

PS: Most Swiss speak german too ;)

**Comment by hungerburg** [comment]
Further evidence that the Swiss need not special minority protection - they know the word, see for example this very official looking document, where they write "Gehweg" quite often - https://www.astra.admin.ch/dam/astra/de/dokumente/langsamverkehr/velos_auf_trottoirs-entscheidungshilfedervelokonferenzschweiz200.pdf.download.pdf/velos_auf_trottoirs-entscheidungshilfedervelokonferenzschweiz200.pdf

**Comment by hungerburg** [comment]
Ich hab schon einmal die DE:tag:footway Wiki Seite aktualisiert, das ist nun auch viel leserlicher, OSM muss nicht so gestelzt klingen, dass einem Beamten die Scham kommt dabei.

**Comment by manfredbrandl** [comment]
There are so many aspects ... Gehweg is - according to Wikipedia - the part of the street intended for Walking, also called Bürgersteig or Gangsteig/Gehsteig/Trottoir in DE/AT/CH. Fußweg is - also according to Wikipedia - more general and includes also all other contructions for walking. There are many way for walking and biking and "Fuß- und Radweg" seems to be the most common word for that. If you want to discuss further privately in german we may use another medium.

**Comment by hungerburg** [comment]
I was tired then and forgot to switch to English mode. Anyways, the official wording is simple, clear and concise. The Wiki now reflects that and even mentions the (thanks to you correctly spelled, helveticism) swiss peculiarity. If iD wants to stay apart, let us have the mapping crowd heal it. Happy so far I close the issue. Peter

**Comment by manfredbrandl** [comment]
Hi Peter, the most important part ot the issue "Mehrzweck-Weg" is done in transifex,  and will be solved for iD with the next version 2.18.0 coming to openstreetmap.org. Having small differences between Gehweg, Fußweg und Fussweg will not harm community.

**Comment by hungerburg** [comment]
Yes, the heart of the issue is fixed. I brought that up, because I repeatedly had to explain to newcomers, that the hiking path they created was truly awesome, yet it was not a "Fußweg" but a "Mehrzweck-Weg" instead.

I am afraid that this first step will not help all that much, as long as there is still the very broad meaning "Fußweg" top in the options and I will still have to apologize that the confusing naming is not my fault when I comment their changeset and they explain to me, that of course the hiking path is a "Fußweg/footway".

PS: To recap your google hits counting argument against "Gehweg": A smaller number of hits for "Gehweg" actually speaks for its advantage, less hits, for something that it less common, the other way around would be a sign for something that is wrong.

PPS: I think the origin is just sloppy translation, like german people who say "undertaker" for "Unternehmer".

**Comment by manfredbrandl** [comment]
I think its worthwile to further optimize German preset-texts in iD. In iD 3 one will select the feature type before mapping, but in iD 2 it's the other way round. After mapping a line one can see "Größere Straßen...", "Nebenstraßen...", "Schienen..." and "Wege..." with a walking person: 
![image](https://user-images.githubusercontent.com/5360273/81913337-1f5ac500-95d0-11ea-91ba-8474ed3e79a2.png)
Klicking on "Wege..." one gets (after using the new translation in ID 2) "Weg für nichtmotorisierten Verkehr", "Fußweg", "markierter/gekennzeichneter Übergang", "Bürgersteig/Gehsteig/Trottoir", "Treppe", "Radweg", "Reitweg" and "Fußgängerstraße". Please try yourself and use the "i" to see the explantion from the wiki. Suggestions for further optimizations are warmely welcome.

**Comment by hungerburg** [comment]
Hello Manfred, my wish for iD is, to enable newbies to get it right on first try. I consider that important for the whole OSM project, because iD sure is the number on entry point for newcomers. I just happen to care. Editing the wiki I learned, that, while the same circumstance can be mapped in diverse ways, there are competing schools advocating the single right way and that there are quite fierce scholars.

The first paragraph above sounds very intuitive. The paragraph below the icon gets messy a bit. Not the least, because the world is a bit messy and perhaps inevitably so the OSM concepts too. I'd say, people will avoid anything that sounds complicated or requires long explanations, so I'd rephrase and resort the list:

"Weg", "Radweg", "Reitweg", "Gehweg (Fussweg)", "Bürgersteig (Gehsteig/Trottoir)", "markierter Übergang", "Treppe", …

Don't know if that helps - ordered like that, it should at least please the school of "bare path + access" school of truth

PS: Gehweg deliberately moved back so its close to Gehtsteig, where it belongs :)

**Comment by hungerburg** [comment]
Hello Manfred, did you get anywhere? Here a fresh newbie _Fußweg_ https://www.openstreetmap.org/way/803294117 - Perhaps it it easier to change the meaning of footway in OSM instead?
